owid/walden/index/who/2020-12-01/ghe.json

```diff
@@ -0,0 +1,16 @@
+{
+  "namespace": "who",
+  "short_name": "ghe",
+  "name": "Global Health Estimates - World Health Organization (2020)",
+  "description": "WHO\u2019s Global Health Estimates (GHE) provide the latest available data on death and disability globally, \nby region and country, and by age, sex and cause. The latest updates include global, regional and country \ntrends from 2000 to 2019 inclusive. By providing key insights on mortality and morbidity trends, these estimates\nare a powerful tool to support informed decision-making on health policy and resource allocation.\n",
```

Comment:
I think the `\n` will not be parsed as line breaks (you can try). In any case it's better to not have explicit line breaks in between long sentences (and simply let the sentences be long in the metadata file).

Reply:
> Do you mean that the source data has changed? In that case, it makes sense to create a new version of the ingest code in walden, and new walden files. That way we can understand the data that we used to have in production (even if the old ingest code may not work anymore). Do you agree?

Yeh the original API point is no longer available and the current one has more data, although I don't think it's a new version, just a broader release of the same dataset.  

In terms of the date used in the version folder, for the original dataset I think we used the date of ingest - would it make sense to do the same in this ingest. I know it's mean to be the date the data is published, but in that case, this new upload would be `2020-12-01` and therefore before the existing update. What do you think?

Reply:
I'd say this still means that the source data changed, so I'd call it a different version.
I think it's good to have the current date as `version` (that's when you are creating the current version of the dataset), and `publication_date` will be the date when the source published the data (which can be whatever date). I think this criterion is safer than using `publication_date` as the `version` (because sometimes the source may change things, as it happened now, but still claim that it's "the same data", hence with the same publication date).

Reply:
Ah okay -  I didn't realise `version` was an option, thanks! Have added that as the current date.